manifests/f/Funkin'Team/FridayNightFunkin'/v2.8/Funkin'Team.FridayNightFunkin'.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Funkin'Team.FridayNightFunkin'
+PackageVersion: v2.8
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: msi
+  InstallerUrl: https://github.com/Muhammed87643/Friday-Night-Funkin-Silent-Installer/releases/download/v1.0/Funkin.msi
```

Comment:
This installer URL doesn't seem like it comes from the original publisher. If this is a repackaged application, it cannot be accepted
![image](https://user-images.githubusercontent.com/12611259/221687702-f03bb2b5-395d-4b1d-946d-366a48adb006.png)
![image](https://user-images.githubusercontent.com/12611259/221688006-24b328b1-e7a9-419a-a386-781bef524170.png)